### PR TITLE
Added JumpVariable and NoiseVariable

### PR DESCRIPTION
**Pull request by Vaibhavdixit02**

### CODE REVIEW

**Comment by ChrisRackauckas** [comment]
Needs tests.

**Comment by Vaibhavdixit02** [comment]
@ChrisRackauckas basic test have been added

File: test/runtests.jl [hunk 1]
```diff
@@ -57,3 +59,12 @@ u = DependentVariable(:u, [1])
 # Derivatives
 dsin = D*sin(t)
 isequal(expand_derivatives(dsin),cos(t))
+
+#SDE
+eqs = [D*x == σ*(y-x),
+       D*y == x*(ρ-z)-y,
```
Comment (ChrisRackauckas) [inline test/runtests.jl]: you didn't add the SDE or jump portions in here but labelled it SDE?

**Comment by Vaibhavdixit02** [comment]
Fixed, sorry!

**Pull request merged**